main.py

```
@@ -176,7 +179,7 @@ def open_reader(self, comic: dict):
         cont = ReadingController(comic)
         cont.read_comic()
         
-    def print_hi(self):
+    def print_hi(self, comic):
```

Ruff (0.11.9)

Missing return type annotation for public function `print_hi`

Add return type annotation: `None`

(ANN201)